### ISSUE TITLE
Cryptopia.js not working

### ISSUE THREAD

**Issue by Popcaughan** [ManuCart/Cryptocurrency-Portfolio#11]
Hi, I get "{Error=Signature does not match request parameters., Success=false}" in the logs when I try to run the cryptopia() method. 

Could it be because there is a + and an = in my secret like this:

j4Ni6YNpzp9TmcKu3dhy32HkKw7SR6lRT+BXq6D4lZiU=

These are quite common in Cryptopia sectets.

Here's the whole log:

[18-01-09 14:54:31:913 PST] mZFLkyvTelC5g8XnyQrpOw==  mZFLkyvTelC5g8XnyQrpOw==
[18-01-09 14:54:31:914 PST] https%3a%2f%2fwww.cryptopia.co.nz%2fapi%2fgetbalance
[18-01-09 14:54:31:914 PST] signature b08ce14d2e9c4956ad4a2bee3ba8f4bcPOSThttps%3a%2f%2fwww.cryptopia.co.nz%2fapi%2fgetbalance1515538471mZFLkyvTelC5g8XnyQrpOw==
[18-01-09 14:54:31:995 PST] signed: qMf1ONALXtokfN50jPMHqzBv084fH+RjL+AcDzh4VtI=
[18-01-09 14:54:31:996 PST] header_value: amx b08ce14d2e9c4956ad4a2bee3ba8f4bc:qMf1ONALXtokfN50jPMHqzBv084fH+RjL+AcDzh4VtI=:1515538471
[18-01-09 14:54:32:092 PST] {Error=Signature does not match request parameters., Success=false}
[18-01-09 14:54:32:093 PST] []

Thanks.

**Comment by ManuCart** [comment]
Hi,
It's work for me.
If I remenber you need to use local settings United States.
Go to your sheet
File -> Spreadsheet settings -> tab General -> Locale : United States
Reload and tell me if it's working
If it's true I will add this tips in Readme.md

**Comment by Popcaughan** [comment]
Hi, I have updated my locale and set my timezone to EST but was still getting the same error.

So I requested a new key / secret combination until I managed to get one which didn't have a +, / or any other special characters in it. 

Using that key / secret combination, the code is now working :)

So you might want to add a warning about cryptopia api keys in the readme.md

Thanks for your help, loving your code!
